README.md

```diff
@@ -30,7 +31,7 @@ fast in-memory access to the webpack assets.
 7. [Attribution](#attribution)
 8. [License](#license)
 
-## Getting Started
+<h2 align="center">Getting Started</h2>
```

Comment:
We should avoid using `html`, it is bad practice. Also let's do:
- Remove `Maintainers` section (it is outdated) and no need
- Don't move `div` after description

Reply:
> * Don't move `div` after description

sorry, I want to get more details.

Reply:
@hiroppy yep L21